raylib.nelua

```diff
@@ -43,7 +43,7 @@ global Matrix     <cimport, nodecl> = @record{
    m2: float32, m6: float32, m10: float32, m14: float32,
    m3: float32, m7: float32, m11: float32, m15: float32
 }
-global Color     <cimport, nodecl> = @record{r: cuchar, g: cuchar, b: cuchar, a: cuchar}              -- Color type, RGBA (32bit)
+global Color     <cimport, nodecl> = @record{r: uint8, g: uint8, b: uint8, a: uint8}              -- Color type, RGBA (32bit)
```

Comment:
Forgot to comment about this, it's guaranteed that `uint8` is equivalent for `cuchar`, changing to `uint8` allows us to use _u8 literal for colors, like 255_u8 was used in an example, this way avoiding other type casting